rackcity/views/rackcity_utils.py

```diff
@@ -177,6 +177,11 @@ def __init__(self, *args, **kwargs):
         Exception.__init__(self, *args, **kwargs)
 
 
+class PowerConnectionException(Exception):
```

Comment:
I want to put custom exceptions in their own file at some point but it's low priority

Reply:
good idea